entries/grub-glide/game.json

```diff
@@ -0,0 +1,25 @@
+{
+    "developer": "Ethan Hazel",
+    "files": [
+        {
+            "default": true,
+            "filename": "GrubGlide.gb",
+            "playable": true
+        }
+    ],
+    "license": "Zlib",
+    "platform": "GB",
+    "repository": "https://github.com/Exetric/GrubGlide",
+    "screenshots": [
+        "1.png",
+        "2.png",
+        "3.png",
```

Comment:
I think this trailing comma should be removed?

Reply:
Just fixed the issue, thanks for helping point it out!